pkgs/servers/sql/postgresql/ext/pgml/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib
```

Comment:
Another Alejandrism?

Reply:
Haha, maybe. I dont care too much about the formatting, if you let me know how you prefer it, im happy to change.

Reply:
Ive opened this as draft because it still doesnt compile properly. I need to test a few things, this PR might get noisy with commits (just letting you know )

But thanks for the suggested changes, ill be sure to integrate them !